Files/UserControls/SidebarControl.xaml

```diff
@@ -211,6 +216,126 @@
+                                                <VisualState.Setters>
+                                                    <Setter Target="LayoutRoot.Background" Value="{ThemeResource ToggleButtonBackgroundCheckedPointerOver}" />
+                                                    <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource ToggleButtonForegroundCheckedPointerOver}" />
+                                                </VisualState.Setters>
+                                            </VisualState>
+                                            <VisualState x:Name="CheckedPressed">
+                                                <VisualState.Setters>
+                                                    <Setter Target="LayoutRoot.Background" Value="{ThemeResource ToggleButtonBackgroundCheckedPressed}" />
+                                                    <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource ToggleButtonForegroundCheckedPressed}" />
+                                                </VisualState.Setters>
+                                            </VisualState>
+                                            <VisualState x:Name="CheckedDisabled">
+                                                <VisualState.Setters>
+                                                    <Setter Target="LayoutRoot.Background" Value="{ThemeResource ToggleButtonBackgroundCheckedDisabled}" />
+                                                    <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource ToggleButtonForegroundCheckedDisabled}" />
+                                                </VisualState.Setters>
+                                            </VisualState>
+                                        </VisualStateGroup>
+                                    </VisualStateManager.VisualStateGroups>
+
+                                    <Border Width="{TemplateBinding MinWidth}">
+                                        <Viewbox
+                                            x:Name="IconHost"
+                                            Width="18"
+                                            Height="18"
+                                            HorizontalAlignment="Center"
+                                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                                            AutomationProperties.AccessibilityView="Raw">
+                                            <TextBlock
+                                                x:Name="Icon"
+                                                Text="&#xEAD6;"
+                                                FontSize="{TemplateBinding FontSize}"
+                                                FontFamily="{StaticResource OldFluentUIGlyphs}"
+                                                AutomationProperties.AccessibilityView="Raw"/>
```

Comment:
Can you change this to a FontIcon and use this glyph `&#xE700;`?

Reply:
@yaichenbaum the template I've copied do use this glyph, I modified it to match the other icons of the sidebar more. In case using glyph `&#xE700;` is more important, it might be better to remove this template override and have the default icon. What do you think?

PaneToggleButtonStyle of NavigationView:

https://github.com/microsoft/microsoft-ui-xaml/blob/7b55398026652cc95e82d96dc5be1aae3ee1204f/dev/NavigationView/NavigationView_rs1_themeresources.xaml#L281-L368

Reply:
The glyph I mentioned above will use the best font family available for the version of Windows the user is running, so I think it's best to keep the template and switch to the new icon.